packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -716,3 +744,18 @@ bool defaultSchedulingStrategy({ int priority, SchedulerBinding scheduler }) {
     return priority >= Priority.animation.value;
   return true;
 }
+
+/// Performance information for a Flutter frame.
+class FrameInfo {
+  FrameInfo(this.number, this.startTime, this.elapsed);
+
+  /// The frame number.
+  final int number;
+  /// The start time in microseconds.
```

Comment:
"start time in microseconds" only raises more questions! relative to what? is it monotonic? is it affected by timeDilation? etc etc etc